_docs/NumberInput/NumberInput.md

```diff
@@ -0,0 +1,96 @@
+# NumberInput Component
+
+**NumberInput** is a form-element component that enables inputting values of the type number. The values can be confined to a specific range, and the "step" (or increment) between the values can be defined.
+
+![NumberInput examples](./assets/numberInput.png)
+
+The NumberInput component improves upon the native `<input type="number">`:
+
+* provides ability to customize the stepper arrows design
+* provides a common React+Typescript API
+* works out the kinks of native implementations
+
+## Component API
+
+> tbd :: add method definition
+
+### Component Props
+
+| name        | type                                  | defaultValue | isRequired | description                              |
+| ----------- | ------------------------------------- | ------------ | ---------- | ---------------------------------------- |
+| value       | number |              | yes        | Used to set and change the value of the input |
```

Comment:
`value` isn't required, because it and others are controlled/uncontrolled depends on props.
@VKobeliatsky can explain if needed, here is all props: https://github.com/wix/stylable-components/blob/volodymyrk/31-number-input-uncontrolled/src/components/number-input/number-input.tsx#L10-L23

check `defaultValue `